pages/servers/[serverId]/polls/results/[pollId].tsx

```diff
@@ -27,10 +29,20 @@ const PollResults: NextPage = () => {
 
   const isLoadingPoll = !pollData && !error
 
+  // TODO: clean up comments.
   return (
     <Box bg='dark-2' minH='calc(100vh - 90px)' pt='4rem' pb='8rem'>
       <Box bg='dark-1' maxW='2xl' mx='auto' p='2rem 3rem'>
-        <Govcrumb />
+        {/* <Govcrumb currentServerName={currentServer?.name}/> */}
+        <Breadcrumb
+          spacing='8px'
+          separator={<ChevronRightIcon color='gray.500' />}
+          color='gray.300'
+        >
+          <BreadcrumbItem>
+            <BreadcrumbLink href={`/servers/${currentServer?.id}`}>Back</BreadcrumbLink>
```

Comment:
Hmm optional chaining might return a weird route? Maybe a ternary might work more neatly. If there is no server ID, should just route to /servers

Reply:
I think we can do:

```suggestion
{`/servers/${currentServer?.id ?? ''}`}
```

Reply:
I'll fix it, regardless.